tests/common.py

```diff
@@ -973,9 +973,11 @@ def last_call(self, method=None):
             return None
 
 
-class MockConfigEntry(config_entries.ConfigEntry):
+class MockConfigEntry(config_entries.ConfigEntry[_DataT]):
     """Helper for creating config entries that adds some defaults."""
 
+    runtime_data: _DataT
```

Comment:
Tests should not interact with config entry runtime data as that's an integration detail. Adding this attribute here is an antipattern.

Reply:
It is high time to review the unifi tests, I will fix

Reply:
@MartinHjelmare I have now reworked enough tests to get rid of the runtime_data dependency 

https://github.com/home-assistant/core/pull/119202